client/cmd/dexcctl/sample-dexcctl.conf

```diff
@@ -0,0 +1,21 @@
+[Application Options]
```

Comment:
Not really sure we want this line.  It's just confusing if it is not required.  Any advice from @davecgh or @jrick on this would be appreciated.

Reply:
hate these things